video/vaapi.c

```diff
@@ -677,30 +679,36 @@ static void drm_destroy(void *native_ctx)
     talloc_free(ctx);
 }
 
-static void drm_create(VADisplay **out_display, void **out_native_ctx)
+static void drm_create(struct mpv_global *global, struct mp_log *log,
+        VADisplay **out_display, void **out_native_ctx)
 {
-    static const char *drm_device_paths[] = {
-        "/dev/dri/renderD128",
-        "/dev/dri/card0",
-        NULL
-    };
+    char *device = NULL;
+    mp_read_option_raw(global, "vaapi-device", &m_option_type_string,
+            &device);
```

Comment:
You must free the string returned by this.

Reply:
fixed (don't know why github didn't actually send the comment last time)